README.md

```
@@ -54,11 +54,38 @@ Get started with socktainer CLI in just a few commands:
-FolderWatcher] Started watching $HOME/Library/Application Support/com.apple.container
+[ INFO ] Docker context 'socktainer' ready — run: docker context use socktainer
+[ NOTICE ] DNS server listening on port 2054
 [ NOTICE ] Server started on http+unix: $HOME/.socktainer/container.sock
 ...
 ```
 
+If Apple Container's own service isn't already running, socktainer starts it
+automatically first (like colima does for Docker) — no separate `container
+system start` step needed — then continues with the rest of startup above:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)